LICENSE.md

```diff
@@ -2,7 +2,7 @@
 Copyright (c) Microsoft Corporation
 All rights reserved. 
 MIT License
-Permission is hereby granted, freeof charge, to any person obtaining a copy of this software and associateddocumentation files (the Software), to deal in the Software withoutrestriction, including without limitation the rights to use, copy, modify,merge, publish, distribute, sublicense, and/or sell copies of the Software, andto permit persons to whom the Software is furnished to do so, subject to thefollowing conditions:
```

Comment:
If you are starting that - please take a look for the same issue later - "withoutrestriction", "modify,merge", "andto", "thefollowing". That's only in this same paragraph.
Even if this file has only 2 more lines - it's screwed also in them multiple times.

I wonder if there was a single review for this file, before publishing it...